xbmc/platform/darwin/tvos/input/LibInputSettings.h

```diff
@@ -0,0 +1,20 @@
+/*
+ *  Copyright (C) 2019- Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#import <Foundation/Foundation.h>
+
+extern float const REPEATED_KEYPRESS_DELAY_S;
```

Comment:
1. both should have type NSTimeInterval
2. why are they defined here, but actually used in another file?

Reply:
Only reason they were thrown in here was they were some magic number hidden away in ther original location. I started to strip anything that may need to be user changable into Settings.

Happy to move out from here, as after the refactor, i dont think it ended up making sense to make it an end user setting.